### PR TITLE
US123226: paging summary

### PR DESCRIPTION
**Pull request by martinwang-d2l**
https://rally1.rallydev.com/#/29180338367d/custom/486232622040?detail=%2Fuserstory%2F463040293896

This story adds a summarizer to the timing and display accordion, to show whether paging options have been applied. The screenshot shows the summarizer when paging is applied: 

Notice that the summarizer is below the timing summarizer (according to field order, as described in story AC). 

EDIT: changed comparison to != 'none', to account for legacy paging option. Also changed lang term to sentence case. 

image

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import '@brightspace-ui/core/components/icons/icon.js';
+import { ActivityEditorMixin } from '../mixins/d2l-activity-editor-mixin';
+import { html } from 'lit-element/lit-element.js';
+import { LocalizeActivityQuizEditorMixin } from './mixins/d2l-activity-quiz-lang-mixin.js';
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { sharedActivityUsageCollection as store } from './../state/activity-usage-collection';
+
+class ActivityQuizPagingSummary
```
Comment (dustxd) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:8]: Can you add a comment similar to what Margaret did here?

https://github.com/BrightspaceHypermediaComponents/activities/pull/2581/files#diff-91cc843b9fc64123ba6c49c7fed5500ae470ba2e1aa04ca668e8c231822d8d4cR8

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	}
+
+	render() {
+		const activityUsageCollection = store.get(this.href);
+		if (!activityUsageCollection) {
+			return html``;
+		}
+
+		const { selectedPagingType } = activityUsageCollection;
+		const pagingSelected = (selectedPagingType === 'pagebreakaftereachsubcollection' || selectedPagingType === 'oneactivityperpage');
```
Comment (dustxd) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:23]: I think we need to display that for legacy as well?

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js (hunk 1 above)
Comment (martinwang-d2l) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:9]: Sure, added

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+		if (pagingSelected) {
```
Comment (dustxd) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:25]: I think it's better if we do 
```
if (!pagingSelected) {
  return html ``;
}

return html`${this.localize('pagingSummary')}`;
```
Looks more consistent with above code and easier to read the logic

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js (hunk 2 above)
Comment (dustxd) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:23]: Reference here: https://github.com/BrightspaceHypermediaComponents/activities/pull/2601

Comment (martinwang-d2l) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js]: Yeah true - fixed

**Review by dustxd — DISMISSED** [review]

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+	render() {
+		const activityUsageCollection = store.get(this.href);
+		if (!activityUsageCollection) {
+			return html``;
+		}
+
+		const { selectedPagingType } = activityUsageCollection;
+
+		if (selectedPagingType === NO_PAGE_BREAK) {
```
Comment (stevenao) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js:26]: will `selectedPagingType` ever be undefined b/c of flag off?

Comment (martinwang-d2l) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-paging-summary.js]: True - added check

**Review by stevenao — APPROVED** [review]

**Pull request merged**